test/test_get.py

```diff
@@ -281,3 +283,64 @@ def test_message_printed_when_no_id_provided(
             mock_output.assert_not_called()
             assert result.exception
             assert result.exit_code == 1
+
+    @patch.object(dataset.Dataset, "output_dataset_details")
+    @patch("dafni_cli.get.get_all_datasets")
+    @patch("dafni_cli.get.check_for_jwt_file")
+    class TestDatasets:
+        """Test class to test the get group datasets command"""
+
+        @patch("dafni_cli.get.process_response_to_class_list")
+        def test_get_all_datasets_called_with_jwt_from_context(
+            self,
+            mock_create,
+            mock_jwt,
+            mock_get,
+            mock_output,
+            processed_jwt_fixture,
+            get_dataset_list_fixture,
+        ):
+            # SETUP
+            # setup get group command
```

Comment:
I don't think this comment describes what happens below it

Reply:
updated